pkg/ast/pipesearch/searchHandler.go

```diff
@@ -454,7 +458,7 @@ func getQueryResponseJson(nodeResult *structs.NodeResult, indexName string, quer
 	httpRespOuter.CanScrollMore = canScrollMore
 	httpRespOuter.TotalRRCCount = numRRCs
 	httpRespOuter.MeasureFunctions = nodeResult.MeasureFunctions
```

Comment:
MeasureResults is regenerated based on node.Histogram, should we also use the new measureFuncs to replace the previous nodeResult.MeasureFunctions?